leftwm/src/bin/leftwm-check.rs

```diff
@@ -50,6 +50,7 @@ async fn main() -> Result<()> {
             }
             config.check_mousekey(verbose);
             config.check_workspace_ids(verbose);
+            #[cfg(feature = "lefthk")]
```

Comment:
We might want to go ahead and add the notice discussed earlier in the thread here?
```rust
#[cfg(not(feature="lefthk"))]
warn!("Ignoring checks on keybinds as you compiled for an external hot key daemon."); // or should this be a warning? idk
```

Reply:
This should be more in line with the rest of the current `leftwm-check` code
```suggestion
            #[cfg(not(feature = "lefthk"))]
            println!("\x1b[1;93mWARN: Ignoring checks on keybinds as you compiled for an external hot key daemon.\x1b[0m");
           #[cfg(feature = "lefthk")]
```